alex-smg/scanup-back
# feat(question): Add CRUD

## How to test
* Import postman document
* If you want to create a new user, move the route for create a new user outside of the middlewares
* Log in and get copy the token
* Then, paste the token in the header (`token`)
* Check if you can Create/Read/Post/Put/Delete a question
